.github/workflows/release.yml

```
@@ -1,107 +1,34 @@
 name: release
-run-name: Release ${{ inputs.VERSION }} (pre-release - ${{ inputs.IS_PRE_RELEASE }}) by @${{ github.actor }} from ${{ github.ref_name }}
+run-name: Release by @${{ github.actor }} from ${{ github.ref_name }}
 
 on:
-  workflow_dispatch:
-    inputs:
-      VERSION:
-        description: "The version to release"
-        required: true
-      IS_PRE_RELEASE:
-        description: "It IS a pre-release"
-        required: true
-        default: false
-        type: boolean
+  push:
+    branches:
+      - develop
+      - feature/*
+      - fix/*
+      - chore/*
```

I use even another pattern for naming branches that are not associated with an issue (yet). I think there is no need to filter for names. We can just simplify and allow all branches.

```suggestion
      - "*"
```

This worked for me for filtering for tags only (other way around) [here](https://github.com/merschformann/gotz/blob/main/.github/workflows/publish.yml#L6) for example. :blush: